src/snakeGame/Mushroom.java

```diff
@@ -0,0 +1,92 @@
+package snakeGame;
+
+import edu.macalester.graphics.GraphicsGroup;
+import edu.macalester.graphics.Ellipse;
+import edu.macalester.graphics.Rectangle;
+import java.awt.Color;
+
+/** This is a class representing the mushrooms, the food the snake will eat to get longer. */
+
+public class Mushroom extends GraphicsGroup {
+    private static final Color MUSHROOM_RED = new Color(237,120,131);
+    private static final Color MUSHROOM_WHITE = new Color(247, 250, 250);
+    private int mushroomWidth;
+    private int mushroomHeight;
+    private int stemWidth;
+    private int stemHeight;
+
+    public Mushroom(int mushroomWidth, int mushroomHeight, int stemWidth, int stemHeight, int x, int y) {
+        this.mushroomHeight = mushroomHeight;
+        this.mushroomWidth = mushroomWidth;
+        this.stemHeight = stemHeight;
+        this.stemWidth = stemWidth;
+        // x and y are set to 0 in makeMushroomHead and makeMushroomStem to avoid conflicting 
+        // coordinates within the GraphicsGroup and main gameLayer
+        makeMushroom(makeMushroomHead(mushroomHeight, mushroomWidth, 0, 0), makeMushroomStem(stemWidth, stemHeight, 0, 0));
+        this.setPosition(x, y);
+    }
+
+    /**
+     * 
+     * @param mushroomHeight
+     * @param mushroomWidth
+     * @param x
+     * @param y
+     * @return A graphics group of a red ellipse with white dots, representing the top half of the 
+     *         mushroom.
+     */
```

Comment:
Be careful not to leave half-filled doc templates like this lying around. Either the method needs documentation or it doesn’t, so complete or delete!